src/index.js

```diff
@@ -150,8 +152,9 @@ class Dotenv {
     // However, if someone targets Node or Electron `process.env` still exists, and should therefore be kept
     // https://webpack.js.org/configuration/target
     if (this.shouldStub({ target, version })) {
+      const replace = this.config.prefix.slice(0, -1) // Remove the dot in the end of prefix
```

Comment:
I'm not sure about this assumption here. Basically it means that the dot _has_ to be at the end of the prefix. But if it has to be there, why not just make it implicit, and remove it from the config option altogether?
i.e. `prefix: 'process.env'` instead of `prefix: 'process.env.'`

Reply:
Thank you for your time for this review! Initially, I tried to account for the case, suggested by @mrsteele:
```js
new Dotenv({
  prefix: `MY_TEST_`
})
// MY_TEST_PORT
```
But I guess for now it's better to stay with implicit `.` separator, as it makes config more readable and less error prone for most use cases.

Reply:
ah, I see. Okay, I'll let @mrsteele make the final call on this one, since it's his package. 👍

Reply:
Yeah I suggest we be as explicit as we can here. I don’t want to back us into a corner.

make the dot explicit please 🙏